plugins/volumeFilter.go

```diff
@@ -222,15 +222,14 @@ func volumeFilterFn(dailyOTB *VolumeFilterConfig, dailyTBBAccumulator *VolumeFil
 
 	// if exact mode and with remaining capacity, update the op amount and return the op
 	// else, return nil
-	remainingCapacity := (cap - otb - tbb) / capPrice
-	if remainingCapacity <= 0 {
+	// TODO DS Determine whether this calculation works for a buy offer.
+	offerAmount = (cap - otb - tbb) / capPrice
```

Comment:
can we call this `newOfferAmount` as a new variable instead of reusing the `offerAmount` variable?